src/library/scala/collection/LinearSeqOptimized.scala

```diff
@@ -117,20 +117,20 @@ trait LinearSeqOptimized[+A, +Repr <: LinearSeqOptimized[A, Repr]] extends Linea
   }
 
   override /*TraversableLike*/
-  def foldLeft[B](z: B)(f: (B, A) => B): B = {
+  def foldLeft[B](z: B)(@deprecatedName('f) op: (B, A) => B): B = {
```

Comment:
most of the other places `foldLeft` is declared in the collections, the name `op` is used. so Janek is just bringing a few outliers in line.